src/current/cockroachcloud/troubleshooting-page.md

```diff
@@ -170,4 +178,4 @@ A hanging or stuck query using CockroachDB {{ site.data.products.serverless }} m
 
 If you've reached your storage or RU limit, you can [increase your resource limits]({% link cockroachcloud/serverless-cluster-management.md %}#edit-your-resource-limits) and then re-run the query.
 
-If you've only reached your RU limit, you can wait until the next billing cycle when [monthly free RUs]({% link cockroachcloud/plan-your-cluster-serverless.md %}#free-vs-paid-usage) become available and then re-run the query.
```

Comment:
@Amruta-Ranade what is the change here?

Reply:
Oh, I think I just deleted an empty last line.